src/Controller/Admin/AddressCrudController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Address;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\DateTimeField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\FormField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\NumberField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+
+class AddressCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Address::class;
+    }
+
+    public function configureCrud(Crud $crud): Crud
+    {
+        return $crud
+            ->setDefaultSort(['id' => 'DESC']);
+    }
+
+    public function configureFields(string $pageName): iterable
+    {
+        return [
```

Comment:
Consider using `yield` rather than returning an array.

In my experience, yielding makes the field configuration easier to manage, e.g. when configuring a field differently depending on context (see https://github.com/itk-dev/aapodwalk_api/blob/c5e5175ada467ed9c61eb3443645142bf731b4f4/src/Controller/Admin/RouteController.php#L41-L53 for an example).

Reply:
As discussed on teams... it do not seam to work when using fieldset or other "non" field elements.